src/app/components/Signup/Signup.tsx

```
@@ -0,0 +1,351 @@
+'use client';
+import { useRouter } from 'next/navigation';
+import { useEffect, useRef, useState } from 'react';
+
+import { signupUser } from '@/app/api/userAuth';
+import { HideToggle } from '../HideToggle';
```

ESLint

[error] 'HideToggle' is defined but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)